### PR TITLE
Cloud overview page updates

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation
This updates the dbt Cloud over view for the UI Refresh beta. 

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-overview.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ id: "cloud-overview"
 ---
 
 
-[dbt Cloud](https://www.getdbt.com/product/) is a hosted service that helps data analysts and engineers productionize dbt deployments. It comes equipped with turnkey support for scheduling jobs, CI/CD, serving documentation, monitoring & alerting, and an Integrated Developer Environment (IDE).
+[dbt Cloud](https://www.getdbt.com/product/) is a hosted service that helps data analysts and engineers productionize dbt deployments. It comes equipped with turnkey support for scheduling jobs, CI/CD, serving documentation, monitoring & alerting, and an integrated developer environment (IDE).
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/cloud-overview.md:7]:
```suggestion
[dbt Cloud](https://www.getdbt.com/product/) is a hosted service that helps data analysts and engineers productionize dbt deployments. It comes equipped with turnkey support for scheduling jobs, CI/CD, serving documentation, monitoring & alerting, and an integrated development environment (IDE).
```

Comment (nghi-ly) [reply website/docs/docs/dbt-cloud/cloud-overview.md:7]: if we want to use `IDE` in the same way the industry is using it, it should be [this (google search)](https://www.google.com/search?q=ide+stands+for&ei=jbTiYsG4MfzNkPIP-r2AwAQ&ved=0ahUKEwiBs6P8-5v5AhX8JkQIHfoeAEgQ4dUDCA4&uact=5&oq=ide+stands+for&gs_lcp=Cgdnd3Mtd2l6EAMyCggAELEDEIMBEEMyBQgAEJECMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQ6BwgAEEcQsAM6BwgAELADEEM6CwgAELEDEIMBEJECOgQIABBDOgsILhCABBDHARCvAToHCC4QgAQQCkoFCDwSATJKBAhBGABKBAhGGABQqgJYtg1giw9oAnAAeACAAW6IAfoGkgEDOS4xmAEAoAEByAEKwAEB&sclient=gws-wiz) unless we are intentionally wanting to dbt-ize/overload the term

**Review by nghi-ly — CHANGES_REQUESTED** [review]
looks great! one small nit

File: website/docs/docs/dbt-cloud/cloud-overview.md (hunk 1 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/cloud-overview.md:7]: @bossnunta do we want to keep this "developer?" I see Drew's [blog post](https://www.getdbt.com/blog/announcing-the-dbt-ide/) uses "developer" so I am inclined to keep it as that but it's your call!

Comment (runleonarun) [reply website/docs/docs/dbt-cloud/cloud-overview.md:7]: @nghi-ly Let's leave this as-is and come back to it.

**Review by nghi-ly — APPROVED** [review]

File: website/docs/docs/dbt-cloud/cloud-overview.md (hunk 1 above)
Comment (bossnunta) [reply website/docs/docs/dbt-cloud/cloud-overview.md:7]: oh shoot -- sorry I didn't review this before -- IDE should stand for integrated **development** environment, not developer. I think the original note was a typo.

**Pull request merged**